temporal/api/cloud/cloudservice/v1/service.proto

```diff
@@ -125,4 +125,41 @@ service CloudService {
             get: "/api/v1/cloud/regions/{region}",
         };
     }
+
+       // Gets nexus endpoints
+    rpc GetNexusEndpoints(GetNexusEndpointsRequest) returns (GetNexusEndpointsResponse) {
+	option (google.api.http) = {
+            get: "/api/v1/nexus/endpoints",
```

Comment:
```suggestion
            get: "/api/v1/cloud/nexus/endpoints",
```

Need to update the prefix to match others in this service (helps routing and may change soon anyways)

Reply:
whoops, fixed